Header header
sensor_msgs/Image image
geometry_msgs/PointStamped pos
